tutorials/save-query-output.md

```diff
@@ -5,4 +5,95 @@ nav_order: 2
 parent: Tutorials
 ---
 
-## Save the Output of a Query
+GlareDB provides a number of options to save the output of your queries,
+including GlareDB tables, temp tables, and views, and external files in a
+variety of formats.
+
+## Creating a GlareDB Table from a File
```

Comment:
**optional**: It might make sense to mention at the end of this section that you can create a table from a query, not just a file (a file is just a query).

Reply:
Could you say more? How would that differ from what is currently in the first part of the section?

Reply:
This comment arose from two facts:

- the how-to is specific to files
- in the copying to local excel file (ie: specific), we mention this at the end: "You can save the data in a variety of other formats, too, including
Parquet, CSV, and Delta."

---

And so I think we might say, just before "For more information, see the page on `CREATE TABLE`.":

You can create a table from any query and not just files.

Here's a somewhat contrived example:

```sql
CREATE TABLE evens (n int) AS (
    SELECT * FROM  generate_series(0, 100, 2)
);
```